docs/reference/checkpoints_and_actions.md

```diff
@@ -136,303 +136,132 @@ example, the `run_results` dictionary from a WarningAndFailureExpectationSuiteCh
 expectation_suite_severity_level" to indicate if the suite is at either a "warning" or "failure" level.
 
 CheckpointResult objects include many convenience methods (e.g. `list_data_asset_names`) that make working with
-Checkpoint results easier. You can learn more about these methods in the documentation for :py:
-class:`~great_expectations.checkpoint.types.checkpoint_result.CheckpointResult`.
-
-**Example CheckpointResult:**
-
-```python
-{
-    "run_id": run_identifier_object,
-    "run_results": {
-        validation_result_identifier_object: {
-            "validation_result": expectation_suite_validation_result_object,
-            "actions_results": {
-                "my_action_name_that_stores_validation_results": {
-                    "class": "StoreValidationResultAction"
-                }
-            }
-        }
-    }
-    "checkpoint_config": my_checkpoint_config_object,
-    "success": True
-}
+Checkpoint results easier. You can learn more about these methods in the documentation for class: `great_expectations.checkpoint.types.checkpoint_result.CheckpointResult`.
+
+Below is an example of a `CheckpointResult` object which itself contains `ValidationResult`, `ExpectationSuiteValidationResult`, and `CheckpointConfig` objects.
+
+#### Example CheckpointResult:
+
+```python file=file=../../../../tests/integration/docusaurus/reference/core_concepts/checkpoints_and_actions.py#L102-L116
 ```
 
-## Checkpoint configuration examples
+## Checkpoint configuration default and override behavior
 
```

Comment:
Can we include a line like: "The code in the following examples is under test and can be found at: [github link to file]"?

Reply:
You are correct that this is missing, but the standard way we have used to communicate this is to add the link at the bottom in "Additional Notes". The link has been added.

Reply:
Great, I think linking to this great work publicly will really help people make best use of the docs